app/utils/refresh_mat_views.py

```diff
@@ -8,8 +8,10 @@ def refresh_mat_view(name, concurrently):
         db.session.flush()
         _con = "CONCURRENTLY " if concurrently else ""
         db.session.execute("REFRESH MATERIALIZED VIEW " + _con + name)
-    except:
-        print("Problem updating view " + name)
+        db.session.execute("commit")
```

Comment:
I _think_ this hack will work to make the materialized view refresh. Otherwise, I may chuck the towel in and revert to a regular view.